setup.py

```diff
@@ -24,15 +24,15 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '1.5.2',
+    'version': '1.6',
```

Comment:
@itkovian please don't change numbering scheme every time.

Reply:
OK, I clearly forgot the .0.